portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,29 +36,25 @@
 
   private List<String> messages = new ArrayList<>();
   private static final String SPACE = " ";
+  private String numberChoiceString = "";
 
   /** Get all comment entities in the datastore and add them to messages. */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("DO GET");
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
Can use newEntityQueryBuilder() and setLimit() to put the number of comments to fetch in the query.

Example: https://cloud.google.com/datastore/docs/concepts/queries#cursors_limits_and_offsets
API documentation: https://googleapis.dev/java/google-cloud-datastore/latest/index.html (search for StructuredQuery)